en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
+
+  Opens the vault `my vault` and creates a new note called `my note`. The note location is determined by your default new note settings.
+
 - `obsidian://new?vault=my%20vault&file=path%2Fto%2Fmy%20note`
-  This opens the vault `my vault`, and creates a new note at `path/to/my note`.
 
-### Parameters
+  Opens the vault `my vault` and creates a new note at `path/to/my note`.
+
+- `obsidian://new?vault=my%20vault&name=my%20note&content=Note%20content%20here`
 
-- `vault` can be either the vault name, or the vault ID[^1]. Same as action `open`.
-- `name` the file name to be created. If this is specified, the file location will be chosen based on your "Default location for new notes" preferences.
-- `file` a vault absolute path, including the name. Will override `name` if specified.
-- `path` a globally absolute path. Works similar to the `path` option in the `open` action, which will override both `vault` and `file`.
-- `content` (optional) the contents of the note.
-- `clipboard` (optional) use of the contents of the clipboard instead of specifying `content`.
-- `silent` (optional) include this parameter if you don't want to open the new note.
-- `append` (optional) include this parameter to append to an existing file if one exists.
-- `overwrite` (optional) overwrite an existing file if one exists, but only if `append` isn't set.
-- `x-success` (optional) see [[#Use x-callback-url parameters]].
+  Creates a new note called `my note` with the content "Note content here".
+
+- `obsidian://new?vault=my%20vault&name=quick%20note&clipboard=1&silent=1`
+
+  Creates a new note called `quick note` with the current clipboard contents, without opening the note.
 
 ## Create or open daily note
 
-The `daily` action creates or opens your daily note. The [[Daily notes]] plugin must be enabled.
+The `daily` action creates or opens your daily note. The [[Daily notes]] plugin must be enabled for this action to work.
+
+### Parameters
+
+The `daily` action accepts the same parameters as the [[#Create note|new]] action.
 
 ### Examples
 
 - `obsidian://daily?vault=my%20vault`
-  This opens the vault `my vault`, and creates or opens the daily note.
 
-### Parameters
+  Opens the vault `my vault` and creates or opens the daily note for today.
+
+- `obsidian://daily?vault=my%20vault&append=%0A%0A%23%23%20Evening%20reflection`
 
-The `daily` action accepts the same parameters as the `new` action.
+  Opens today's daily note and appends two newlines followed by "## Evening reflection".
 
 ## Open search
 
-The `search` action opens [[Search]] in the specified vault, and optionally perform a search term.
+The `search` action opens [[Search]] in the specified vault and optionally performs a search term.
+
+### Parameters
+
+- `vault` — Can be either the vault name or the vault ID[^1].
+- `query` (optional) — The search term to perform. This should be URI-encoded and can include [[Search#Search operators|search operators]].
```

Comment:
Since I'm not too familiar with obsidian search operators: do any of the _operators_ also need to be encoded?

Also URI vs URL?